suseconnect-keepalive.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Run SUSEConnect --keepalive
+Wants=suseconnect-keepalive.timer
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/SUSEConnect --keepalive
+
```

Comment:
Notice that there is a bug open on our side in which we are not considering environment variables. As it has been suggested, adding the following `EnvironmentFile=-/etc/sysconfig/proxy` would do the trick.

Reply:
See bugzilla report here: https://bugzilla.suse.com/show_bug.cgi?id=1200994

Reply:
added